src/main/java/org/example/C03_12__PatternAngledTriangleIncreasingNumber.java

```diff
@@ -0,0 +1,22 @@
+package org.example;
+
+import java.util.Scanner;
+
+public class C03_12__PatternAngledTriangleIncreasingNumber {
```

Comment:
"AngledTriangle" does not make much sense. Better "RightAngledTriangle"

Reply:
done